README.md

```diff
@@ -38,6 +38,16 @@ $ npm install -g grunt-cli
 $ gem install sass
 ```
 
+```bash
+$ npm install -g grunt-cli
+```
+
+* Gulp - You may use Gulp for Live Reload, Linting, and SASS or LESS.
```

Comment:
We should make sure that it is explicitly says that Gulp right now is optional, not mandatory.